### PR TITLE
Normalize all the line endings

### PR DESCRIPTION
**Pull request by hstandaert**
## Description

Change the way GIT handles line endings. Line endings will always be converted to LF on commit
More info: https://help.github.com/articles/dealing-with-line-endings/#platform-all

## Motivation and Context

> nog een probleem
> expects LF en niet CRLF
> got a windows machine
> new line is sowieso CRLF

~ _Stieven D'Hooge_

## How Has This Been Tested?

OSX.

⚠️ i.e. Hasn't been tested on windows

### CODE REVIEW

**Comment by delrueba** [comment]
I'd add it for both repo's.
One day... they will split.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text eol=lf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
```
Comment (meirege) [inline .gitattributes:6]: Maybe add svg, jpeg and gif?

**Review by meirege — DISMISSED** [review]

File: .gitattributes (hunk 1 above)
Comment (hstandaert) [reply .gitattributes:6]: svg isn't binary 😉I did add the other ones, though 👍

**Review by daften — APPROVED** [review]
I don't see any problem with this, but await other reviews.

**Pull request merged**